### DISCUSSION TITLE
Simulation Error

### DISCUSSION THREAD

**Discussion by wmespi** [whittlem/pycryptobot#722] [General]
Hi,

I was wondering if anyone has run into this issue when running the simulation. I have not been able to pinpoint what it is in my config that would be causing the problem. I will attach that below as well.

Issue only seems to trigger when there is a test buy order executed.

William

![image](https://user-images.githubusercontent.com/47674314/194728192-c05920aa-91b6-4296-92f5-21da6e435e0e.png)

{
    "coinbasepro": {
        "api_url": "https://api.pro.coinbase.com",
        "config": {
            "base_currency": "ALGO",
            "quote_currency": "USD",
            "live": 0,
            "smartswitch": 1,
            "verbose": 1,
            "statdetail": 1,
            "sim": "fast"
        },
        "api_key_file": "coinbase.key"
    }
}

**Comment by whittlem** [comment]
Can you try with the beta branch? I've done a pretty major rewrite in version 7.

git reset --hard
git checkout beta
git pull

**Comment by whittlem** [comment]
Try v7.0.20 which was released this evening.
